src/analyze/analyze-security.c

```diff
@@ -834,9 +849,9 @@ static const struct security_assessor security_assessor_table[] = {
                 .weight = 1500,
                 .range = 1,
                 .assess = assess_capability_bounding_set,
-                .parameter = (UINT64_C(1) << CAP_SETUID)|
-                             (UINT64_C(1) << CAP_SETGID)|
-                             (UINT64_C(1) << CAP_SETPCAP),
+                .parameter = (UINT64_C(1) << CAP_SETUID) |
+                        (UINT64_C(1) << CAP_SETGID) |
+                        (UINT64_C(1) << CAP_SETPCAP),
```

Comment:
its still aligned, but the alignment but its only aligned by the continuation indent and the first bit flag goes on the same line. Otherwise the alignment might get odd, eg:
```
.some_unnnecessary_oddly_long_parameter_which_is_extra_long = (UINT64_C(1) << CAP_SETUID)|
                                                              (UINT64_C(1) << CAP_SETGID)|
                                                              (UINT64_C(1) << CAP_SETPCAP),
```